packages/android_intent/test/android_intent_test.dart

```diff
@@ -140,6 +140,52 @@ void main() {
         verifyZeroInteractions(mockChannel);
       });
     });
+
+    group('showChooser', () {
+      test('raises error if the intent has no action', () async {
+        try {
+          androidIntent = AndroidIntent.private(
+            package: 'packageName',
+            componentName: 'componentName',
+            channel: mockChannel,
+            platform: FakePlatform(operatingSystem: 'android'),
+          );
+          await androidIntent.showChooser();
+          fail('should raise a StateError');
+        } on StateError catch (e) {
+          expect(e.message, 'action must not be null or empty string');
+        } catch (e) {
+          fail('should raise a StateError');
+        }
+      });
+
+      test('pass right params', () async {
```

Comment:
I don't think `"pass right params"` is a name that future developers will be able to maintain. Can you please specify what condition you're testing so that there's no confusion in the future? In other words, a developer might wonder "what are 'right' params and what are 'wrong' params?" and "is there only 1 possible set of 'right' params?"